src/node_api.cc

```diff
@@ -282,7 +282,11 @@ class CallbackWrapperBase : public CallbackWrapper {
         v8::Local<v8::External>::Cast(
             _cbdata->GetInternalField(InternalFieldIndex))->Value());
     v8::Isolate* isolate = _cbinfo.GetIsolate();
-    cb(v8impl::JsEnvFromV8Isolate(isolate), cbinfo_wrapper);
+    napi_value result = cb(v8impl::JsEnvFromV8Isolate(isolate), cbinfo_wrapper);
+
+    if (result) {
```

Comment:
Anna just commented on the main PR that node style prefers `if (value != nullptr)`. Also below.